CHANGELOG.md

```diff
@@ -17,7 +17,12 @@ and this project adheres to
 - @cosmjs/utils: Add `isDefined` which checks for `undefined` in a
   TypeScript-friendly way.
 - @cosmjs/stargate: Add missing `{is,}MsgBeginRedelegateEncodeObject`,
-  `{is,MsgCreateValidatorEncodeObject}` and `{is,MsgEditValidatorEncodeObject}`.
+  `{is,MsgCreateValidatorEncodeObject}` and
+  `{is,MsgEditValidatorEncodeObject}`.
```

Comment:
Hmm, still a diff in the old entry.

Can you run `yarn format-text` in the repo root?